.ci/set_tag_version_images_linux.sh

```diff
@@ -20,6 +20,7 @@ sed -i "s/che-devfile-registry:.*/che-devfile-registry:$1/g" ../deploy/kubernete
 sed -i "s/che-postgres:.*/che-postgres:$1/g" ../deploy/kubernetes/helm/che/custom-charts/che-postgres/values.yaml
 sed -i "s/che-keycloak:.*/che-keycloak:$1/g" ../deploy/kubernetes/helm/che/custom-charts/che-keycloak/values.yaml
 sed -i "s/eclipse\/che-server:.*/eclipse\/che-server:$1/g" ../deploy/kubernetes/helm/che/values.yaml
+sed -i "s/eclipse\/che-dashboard:.*/eclipse\/che-dashboard:$1/g" ../deploy/kubernetes/helm/che/values.yaml
```

Comment:
@mkuznyetsov @nickboldt Could you confirm that it's the right place to edit to bump Che dashboard image?
Asking because it does not seem to be the latest
![Screenshot_20201013_110256](https://user-images.githubusercontent.com/5887312/95833003-baa0c380-0d43-11eb-9797-3aaad8181429.png)

Reply:
for  helm values yes, as for deploy_che.sh edit attempt, it should be removed from that script

Reply:
deploy_che.sh is removed in https://github.com/eclipse/che/pull/18093/commits/0c1848605d20a0130e8c9ada6014363f9eef59da